packages/zosfiles/src/api/methods/create/Create.ts

```diff
@@ -189,15 +189,28 @@ export class Create {
                         break;
 
                     case "dsntype":
-                        // Key to create a PDSE.  Only valid value is LIBRARY (as far as I know)
+                        // Key to create a PDSE.
                         switch (tempOptions.dsntype.toUpperCase()) {
+                            case "BASIC":
+                                break;
+                            case "EXTPREF":
+                                break;
+                            case "EXTREQ":
+                                break;
+                            case "HFS":
+                                break;
+                            case "LARGE":
+                                break;
+                            case "PDS":
+                                break;
                             case "LIBRARY":
                                 break;
+                            case "PIPE":
+                                break;
```

Comment:
Why does each of these values have a separate `case`? 
What about something like:
```
const type = tempOptions.dsntype.toUpperCase();
const availableTypes = ["BASIC", "EXTPREF", ...]
if (availableTypes.includes(type)) {
  return;
}
```

Reply:
ECMAScript 2016 Array.prototype.includes() function doesn't appear to be available and it looks like CLI is at ES5. I am not aware of consequences of this decision so would need to defer to the rest of the squad

Reply:
This can be done as an alternative in ES5
```
if (availableTypes.indexOf(type) !== -1) {
  return;
}
```